custom_components/sat/climate.py

```
@@ -922,10 +899,7 @@
             self._minimum_setpoint.calculate(self._coordinator.return_temperature)
 
         # If the setpoint is high and the HVAC is not off, turn on the heater
-        if self._setpoint > MINIMUM_SETPOINT:
-            await self.async_set_heater_state(DeviceState.ON)
-        else:
-            await self.async_set_heater_state(DeviceState.OFF)
+        await self.async_set_heater_state(DeviceState.ON if self._setpoint > MINIMUM_SETPOINT else DeviceState.OFF)
```

Ruff (0.8.0)

`MINIMUM_SETPOINT` may be undefined, or defined from star imports

(F405)